src/components/network/network-panel.html

```diff
@@ -19,8 +19,14 @@
   </div>
   <div id="combined-regulation" class="panel-section">
     <div class="item-title">Combined Regulations</div>
-    <button class="btn btn-default glyphicon glyphicon-refresh btn-update"></button>
-    <input class="form-control input-with-update">
+    <input class="form-control">
+    <div id="combined-input" class="input-checkbox">
```

Comment:
id="input" is sufficient as the parent container id already indicates "combined-regulation".